tests/test_gaussianPsf.py

```diff
@@ -37,6 +37,7 @@
 import lsst.geom
 import lsst.afw.table
 import lsst.afw.fits
+import lsst.afw.image
```

Comment:
Could this be moved to its own commit?

Reply:
This was necessary to make the test run: somehow that import was unnecessary before (likely circular afw-internal dependencies), but is necessary now.